native/src/ffi/utils.rs

```diff
@@ -45,3 +47,13 @@ pub unsafe extern "C" fn okapi_string_free(s: *mut std::os::raw::c_char) {
         }
     });
 }
+
+#[no_mangle]
+pub extern "C" fn okapi_version() -> *const c_char {
+    // TODO - Insert git commit ID
+    const PKG_VERSION: Option<&str> = option_env!("CARGO_PKG_VERSION");
+    let rust_str = CString::new(PKG_VERSION.unwrap()).unwrap();
+    let c_ver = rust_str.as_ptr();
+    std::mem::forget(rust_str);
```

Comment:
This part leaks memory, as there's no way to clear it

Reply:
We call back into `okapi_string_free` to free it client side. Hmmm.